bot/constants.py

```diff
@@ -49,9 +49,10 @@
 GREEN_CHEVRON = "<:greenchevron:418104310329769993>"
 RED_CHEVRON = "<:redchevron:418112778184818698>"
 WHITE_CHEVRON = "<:whitechevron:418110396973711363>"
-PAID_ICON = "<:paid_icon:432872213088305153>"
-FREEWARE_ICON = "<:freeware_icon:432872213088305153>"
-FREE_ICON = "<:free_icon:432872213088305153>"
+PAID_ICON = "<:icon_paid:433750296724897798>"
+FREEWARE_ICON = "<:icon_freeware:433750375959494657>"
+FREE_ICON = "<:icon_free:433750340509368330>"
+LINK_ICON = "<:icon_link:433703566830469133>"
```

Comment:
They've changed again btw
```py
PAID_ICON = "<:icon_paid:433738703475834898>"
FREEWARE_ICON = "<:icon_freeware:433696378389004289>"
FREE_ICON = "<:icon_free:433696334457864201>"
LINK_ICON = "<:icon_link:433738861462421504>"
```

Reply:
Yeah, we're testing some stuff with different bot accounts. these emote ids are from the testing server, and they were pushed so I could test with a different token.

Reply:
This should be resolved once we wrap up testing and figure out what's wrong with the resources pagination.